cms/djangoapps/contentstore/views/course.py

```diff
@@ -562,7 +566,8 @@ def format_library_for_view(library):
         u'rerun_creator_status': GlobalStaff().has_user(user),
         u'allow_unicode_course_id': settings.FEATURES.get(u'ALLOW_UNICODE_COURSE_ID', False),
         u'allow_course_reruns': settings.FEATURES.get(u'ALLOW_COURSE_RERUNS', True),
-        u'optimization_enabled': optimization_enabled
+        u'optimization_enabled': optimization_enabled,
+        u'org_names_list': org_names_list
```

Comment:
@elvinquero Add a trailing comma.
```suggestion
        u'org_names_list': org_names_list,
```